exporter/importables.go

```diff
@@ -817,4 +817,65 @@ var resourcesMap map[string]importable = map[string]importable{
 			return nil
 		},
 	},
+	"databricks_notebook": {
+		Service: "notebooks",
+		Name: func(d *schema.ResourceData) string {
+			name := d.Get("path").(string)
+			if name == "" {
+				return d.Id()
+			} else {
+				name = strings.TrimPrefix(name, "/")
+			}
+			re := regexp.MustCompile(`[^0-9A-Za-z_]`)
+			return re.ReplaceAllString(name, "_")
+		},
+		List: func(ic *importContext) error {
+			notebooksAPI := workspace.NewNotebooksAPI(ic.Context, ic.Client)
+			notebookList, err := notebooksAPI.List("/", true)
+			if err != nil {
+				return err
+			}
+			// TODO: emit permissions for notebook folders if non-default,
+			// as per-notebook permission entry would be a noise in the state
+			for offset, notebook := range notebookList {
```

Comment:
We need to exclude `/Repos` from that list.

Reply:
is it listed in workspace api?... mkay...

Reply:
Yep, just re-checked with databricks cli

Reply:
thanks!